ansible/inventory/openstack.yml

```diff
@@ -0,0 +1,3 @@
+plugin: openstack
+inventory_hostname: 'name'
```

Comment:
What does this `inventory_hostname: 'name'` mean?

It is unclear which hosts are pointed to by these scripts, and who creates those hosts. What I understand is these scripts assume the hosts are already created, and idempotently configure them (ensure keys are configured, ensure git is installed, ...).

Reply:
As in the README, this playbook uses OpenStack dynamic inventory and limit the execution based on configured host groups.

Indeed you got it right, this playbook just configure an already created groups of hosts.

inventory_hostname just make it use instance name instead of instance id (which is ugly UUID). There is a [link to this docs already](https://docs.ansible.com/ansible/devel/plugins/inventory/openstack.html) in the README but I'll add it here too.

Reply:
I added more info in efa9504c7fd927de7c20664571ab09b6fe037207